x-pack/plugin/esql/build.gradle

```diff
@@ -292,3 +292,9 @@ tasks.named('stringTemplates').configure {
     it.outputFile = "org/elasticsearch/xpack/esql/enrich/EnrichResultBuilderForBoolean.java"
   }
 }
+
+for(forbidTask in ['forbiddenApisMain', 'forbiddenApisTest'] ){
+  tasks.named(forbidTask).configure {
```

Comment:
This can be simplified to:

```
tasks.withType(CheckForbiddenApisTask).configureEach {
      signaturesFiles += files('src/main/resources/forbidden/ql-signatures.txt')
}
```

Reply:
nice, I pushed https://github.com/elastic/elasticsearch/pull/107789/commits/2a9d49caf583d036bd0076dba4a6a932a83a0432